### DISCUSSION TITLE
How to change headercell's height?

### DISCUSSION THREAD

**Discussion by D-nagasumi** [rsuite/rsuite-table#326] [Q&A]
Hello.
Now, I want change headercell's height.
So I checked prop and try to change headercell's height like this:

`<HeaderCell height={200}>Name</HeaderCell>`

But this code dosen't work. 

How can I change the headercell's height?
Sincerely.

**Comment by simonguo** [comment]
Use `headerHeight` on the Table to change the height of the header.

```tsx
<Table headerHeight={50} />
```
